webpack.prod.js

```diff
@@ -4,14 +4,13 @@ const path = require('path')
 module.exports = {
   module: {
     rules: [{
-      include: [path.resolve(__dirname, 'src')],
-      loader: 'babel-loader',
-
-      options: {
-        presets: [ "@babel/preset-env" ]
+      use: {
+        loader: 'babel-loader',
+        options: {
+          presets: ['@babel/preset-env']
+        }
       },
-
-      test: /\.js$/
+      test: /\.m?js$/
```

Comment:
what is `m.js`?

Reply:
This config is copied from webpack babel-loader document. `.mjs` is the es module js file extension.
For this project, don't need it. I can update it to ".js"